owncloudComLibrary/build.gradle

```diff
@@ -24,6 +25,8 @@ android {
 
         versionCode = 10000401
         versionName = "1.0.4.1"
+
+        manifestPlaceholders = [appAuthRedirectScheme: 'oc']
```

Comment:
Is this also needed here?

Reply:
Yes, otherwise the project won't compile

Reply:
I tried what that issue suggests but it doesn't work

Reply:
Guess we can not do much more here...